Modules/EMCAL/include/EMCAL/CalibCheck.h

```diff
@@ -50,13 +50,15 @@ class CalibCheck : public o2::quality_control::checker::CheckInterface
   float mBadThresholdFractionGoodCellsSupermodule = 0.;       ///< Bad Threshold used in the fraction Good Cells per Supermodule check
   float mBadThresholdCellAmplitudeSupermoduleCalibPHYS = 10.; ///< Bad Threshold used in the PHYS Cell amplitude (Calib) vs. supermodule ID check
   float mBadThresholdCellTimeSupermoduleCalibPHYS = 10.;      ///< Bad Threshold used in the PHYS Cell Time (Calib) vs. supermodule ID (High gain) check
+  float mBadThresholdChannelsFEC = 200.;                      ///< Bad Threshold used in the Number of Dead/Bad/Dead+Bad per FEC check
```

Comment:
A FEC has up to 32 channels: 2 T-Cards, and each T-Card has 16 channels (2*8 in col-row space). Consequently we consider a FEC as medium if more than half of the channels (16) are bad/dead, and bad if a FEC has at least 2/3 of the channels (24) bad or dead. Please change the default values.